instrumentation/delayed_job/test/test_helper.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'delayed_job'
+require 'delayed_job_active_record'
+
+require 'opentelemetry/sdk'
+
+require 'minitest/autorun'
+require 'rspec/mocks/minitest_integration'
```

Comment:
We've opted for Minitest rather than RSpec in this project. This is the first introduction of a RSpec feature. Is it necessary for the tests or can we mock without it?

Reply:
I tried to use minitest's mocks, it was almost impossible to use and required installing 3+ extra gems. For mocks specifically I strongly recommend we use RSpec's mocks which are [compatible with minitest](https://relishapp.com/rspec/rspec-mocks/docs/outside-rspec/integrate-with-minitest)

Reply:
Out of curiosity which tests were you having trouble mocking?  I pretty exclusively used Rspec prior to working on this repo, but haven't found anything that doesn't have some sort of equivalent in minitest.